client/components/findStar.jsx

```diff
@@ -15,12 +16,17 @@ class FindStar extends Component {
         fetch('http://localhost:5000/list')
         .then((res) => res.json())
         .then((data) => {
+            console.log(data);
             this.setState({Celebrity:data}, () => console.log(this.state.Celebrity));
         }
     )}
 
     handleClick(event){
         this.setState({click: event.target.id, isLoaded: true}, () => console.log(this.state.click));
+        this.state.Celebrity.map(celeb => 
+            {if(celeb.name === event.target.id) 
+                this.setState({imgPath: celeb.imgPath});
+            })
```

Comment:
Gosh. please change the way you write if case. Linter should capture this as well.
```
        this.state.Celebrity.map(celeb => {
                if (celeb.name === event.target.id) this.setState({imgPath: celeb.imgPath});
        })
```
or 
```
        this.state.Celebrity.map(celeb => {
                if (celeb.name === event.target.id) {
                        this.setState({imgPath: celeb.imgPath});
                }
        })
```

Reply:
incorrect